config/crd/kustomization.yaml

```diff
@@ -7,15 +7,16 @@ resources:
 - bases/toolchain.dev.openshift.com_masteruserrecords.yaml
 - bases/toolchain.dev.openshift.com_notifications.yaml
 - bases/toolchain.dev.openshift.com_nstemplatetiers.yaml
+- bases/toolchain.dev.openshift.com_socialevents.yaml
+- bases/toolchain.dev.openshift.com_spacebindings.yaml
 - bases/toolchain.dev.openshift.com_spaces.yaml
 - bases/toolchain.dev.openshift.com_templateupdaterequests.yaml
 - bases/toolchain.dev.openshift.com_tiertemplates.yaml
 - bases/toolchain.dev.openshift.com_toolchainclusters.yaml
 - bases/toolchain.dev.openshift.com_toolchainconfigs.yaml
-- bases/toolchain.dev.openshift.com_socialevents.yaml
 - bases/toolchain.dev.openshift.com_toolchainstatuses.yaml
 - bases/toolchain.dev.openshift.com_usersignups.yaml
-- bases/toolchain.dev.openshift.com_spacebindings.yaml
+- bases/toolchain.dev.openshift.com_usertiers.yaml
```

Comment:
and don't forget to modify the PROJECT file https://github.com/codeready-toolchain/host-operator/blob/master/PROJECT and also the CSV: https://github.com/codeready-toolchain/host-operator/blob/master/config/manifests/bases/host-operator.clusterserviceversion.yaml
forgot about that in the last review yesterday

Reply:
Done in https://github.com/codeready-toolchain/host-operator/pull/632/commits/2a861ab3aabe7a28555b76daa2d40a1df5d6c572

Should we also add them to https://github.com/codeready-toolchain/api/blob/master/.github/pull_request_template.md so it's less likely we forget in the future?

Reply:
It looks like SpaceBinding is missing from the CSV? Should it be added there as well?

Reply:
> Should we also add them to https://github.com/codeready-toolchain/api/blob/master/.github/pull_request_template.md so it's less likely we forget in the future?

good idea :+1: 

> It looks like SpaceBinding is missing from the CSV? Should it be added there as well?


whoops :grimacing: yeah, please add it there as well. Good catch :+1: